src/users/UserItem.js

```diff
@@ -40,16 +41,33 @@ export default function UserItem(props: Props): Node {
   } = props;
   const _ = useContext(TranslationContext);
 
-  const user = useSelector(state => getUserForId(state, userId));
+  const user = useSelector(state => tryGetUserForId(state, userId));
 
-  const isMuted = useSelector(getMutedUsers).has(user.user_id);
-  const userStatusEmoji = useSelector(state => getUserStatus(state, user.user_id)).status_emoji;
+  const isMuted = useSelector(getMutedUsers).has(userId);
+  const userStatusEmoji = useSelector(
+    state => user && getUserStatus(state, user.user_id),
+  )?.status_emoji;
 
-  const handlePress = useCallback(() => {
-    if (onPress) {
-      onPress(user);
-    }
+  // eslint-disable-next-line no-underscore-dangle
+  const _handlePress = useCallback(() => {
+    invariant(user, 'Callback is used only if user is known');
+    invariant(onPress, 'Callback is used only if onPress provided');
+    onPress(user);
   }, [onPress, user]);
+  const handlePress = onPress && user ? _handlePress : undefined;
+
+  let displayName;
+  let displayEmail;
+  if (!user) {
+    displayName = _('(unknown user)');
```

Comment:
This is a new UI string; we should add it to messages_en.json.

Reply:
Yeah. I actually did initially add it there, and then got a warning about a duplicate key :)